pkg/proxy/userspace/proxier.go

```diff
@@ -26,14 +26,17 @@ import (
 	"sync/atomic"
 	"time"
 
+	libcontainersystem "github.com/opencontainers/runc/libcontainer/system"
```

Comment:
ditto, this should be `"github.com/opencontainers/runc/libcontainer/userns"`

Reply:
updated